website/static/resource/studio/documentation/query.json

```diff
@@ -0,0 +1,24 @@
+{
+  "entries": {
+    "dsl-service.grammar.parser": {
```

Comment:
Let's not duplicated docs like this, instead, move these service-related entries out (also from studio.json) to its own file, something like `dsl-service.json`. We will have all applications point at these. In studio corresponding change, you need to add this doc endpoint as well